build/webpack.base.conf.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
       'vue$': 'vue/dist/vue.esm.js',
       '@': resolve('src'),
       '@nm': resolve('node_modules'),
+      'zoolander$': 'zoolander/dist/css/derek.css'
```

Comment:
By doing this (along with changes in a couple of other files), we can now exclude `zoolander` from the dist artifact, which is important since it's already loaded on rackspace.com